### ISSUE TITLE
Nudge randomly displays it's window buttons

### ISSUE THREAD

**Issue by macbm** [macadmins/nudge#134]
During our testing of Nudge, we've noticed on multiple Macs that the Nudge sometimes randomly displays the window buttons on the top-left corner of the app. This allows the user to close Nudge with the red Close button. 
In some sessions of having Nudge launched with the buttons on the top-left corner, if we wait a few seconds/minutes and put Nudge behind an active window, the app seems to auto correct it's UI and displays the (?) button correctly.

We are using the default LaunchAgent except we added the -json-url option. We have not tried to switch to a local com.github.macadmins.Nudge.json file, not sure if that matters.

Other than this weird anomaly, Nudge is working great so far.

<img width="1305" alt="Screen Shot 2021-02-25 at 5 05 04 PM" src="https://user-images.githubusercontent.com/38539306/109240726-da81d280-778c-11eb-824d-c5a20c7da9dd.png">

**Comment by erikng** [comment]
Would you be willing to send your json and launch agent?

**Comment by macbm** [comment]
Sure, I'll I can send it to you via Slack, thanks!

**Comment by macbm** [comment]
Also, we'vee noticed is that even when using the example Nudge_Launch_Agent-1.0.0.pkg on github with a configuration profile, results with the buttons on the Nudge window are being displayed upon the installation of the pkg, I seem to be able to replicate it consistently. These are M1 Macs, but I don't think that matters.

**Comment by erikng** [comment]
I think I have fixed the issue: https://github.com/macadmins/nudge/commit/5967f16136ccceb816b33d73a95d58ee26f4f649

After working with @macbm I discovered that this only happens when one key is set: `randomDelay`. We have an asynchronous thread that controls the UI and I didn't realize that this thread just sits and sits, which is why the buttons eventually disappear. I redid the logic and now it's doing what I had initially envisioned.
